### PR TITLE
feat: added methods for the latest endpoints

### PR DESCRIPTION
**Pull request by darkoatanasovski**
## OVERVIEW

added new endpoints for JW ott web application

## WHERE SHOULD THE REVIEWER START?

_e.g. `/src/Models/Account.js`_

## ANY NEW DEPENDENCIES ADDED?

_List any new dependencies added._

- [ ] Does it work in IE >= 11?
- [ ] _Does it work in other browsers?_

## CHECKLIST

_Be sure all items are_ ✅ _before submitting a PR for review._

- [ ] Verify you bumped the lib version according to [Semantic Versioning Standards](http://semver.org)
- [ ] Verify you updated the CHANGELOG
- [ ] Verify this branch is rebased with the latest master

### CODE REVIEW

File: src/constants/index.ts [hunk 1]
```diff
@@ -1,25 +1,33 @@
...
-  requestDataCaptureNoAuthAccess: '/v2/accounts/customers/data-capture',
-  externalAccount: (integration: string): string => `/v2/accounts/external/${integration}`,
+  sendPinCode: "/v2/accounts/pin-codes/send",
+  validatePinCode: "/v2/accounts/pin-codes/validate",
+  requestDataCaptureNoAuthAccess: "/v2/accounts/customers/data-capture",
+  externalAccount: (integration: string): string =>
+    `/v2/accounts/external/${integration}`,
+  getFavorites: "/v2/accounts/media/favorites",
+  getFavorite: (id: string): string => `/v2/accounts/media/favorites/${id}`,
+  getWatchlistHistory: "/v2/accounts/media/watchlist-history",
```
Comment (kmitrovv) [inline src/constants/index.ts]:
```suggestion
  getWatchHistory: "/v2/accounts/media/watchlist-history",
```

**Review by kmitrovv — DISMISSED** [review]
Just check the quotes formatting. Should remain as it was.

**Comment by Jasminna** [comment]
There are lot of formatting lint errors...

**Review by mirovladimitrovski — DISMISSED** [review]

File: src/endpoints/account.ts [hunk 2]
```diff
@@ -1059,29 +1094,288 @@ class Account extends BaseExtend {
...
+   * 	}],
+   * 	"cursor": "https://services.inplayer.com/v2/accounts/media/watchlist-history?cursor=Ksm34SUn3j23j3"
+   * }
+   * ```
+   */
+  async getWatchHistory({
+    filter = 'currently_watching',
+    cursor = '',
+  }: CollectionWithCursorArgs): Promise<
+    AxiosResponse<CollectionWithCursor<WatchlistHistoryData>>
```
Comment (Jasminna) [inline src/endpoints/account.ts]:
```suggestion
    AxiosResponse<CollectionWithCursor<WatchHistoryData>>
```

**Review by mirovladimitrovski — DISMISSED** [review]

**Review by Jasminna — DISMISSED** [review]
Just check the suggested change.
It's not a big deal, but it's better to have similar name with the method :)

**Review by mirovladimitrovski — APPROVED** [review]

**Review by Jasminna — APPROVED** [review]

**Pull request merged**